Automattic/woocommerce-payments
# Fix initial isLoading for deposits overviews data.

Fixes https://github.com/Automattic/woocommerce-payments/issues/5812.

#### Changes proposed in this Pull Request

Inspired by https://github.com/Automattic/woocommerce-payments/blob/e7631f174670aad95e026eb06e2a442279a2033f/client/checkout/blocks/upe-fields.js#L38, we can assume if hasFinishedResolution() returns false, that means the selector has not even started resolving.

I found https://github.com/google/site-kit-wp/issues/2436 as well after googling 'hasFinishedResolution'.

#### Testing instructions

* Enable the `_wcpay_feature_simplify_deposits_ui` feature flag by running `update_option( '_wcpay_feature_simplify_deposits_ui', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/).
* Run `npm run build:client`.
* Go to Payments → Overview.
* With base branch, the Account Balances and Next Deposit components show a number very briefly before the loading state. With PR branch, they should show the loading state immediately.

-------------------


**Post merge**
